piv/key.go

```diff
@@ -210,9 +222,38 @@ func parseAttestation(slotCert *x509.Certificate) (*Attestation, error) {
 			return nil, fmt.Errorf("parsing extension: %v", err)
 		}
 	}
+
+	slot, _ := parseSlot(slotCert.Subject.CommonName)
```

Comment:
nit: for readability, don't ignore the return value even if it's the same result

```
slot, ok := parseSlot(slotCert.Subject.CommonName)
if ok {
    a.Slot = slot
}
```